src/artist-selection/artists-slider.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {ChangeEvent, useContext, useState} from "react";
+import {observer} from "mobx-react";
+import {LocaleContext} from "../spotifun";
+import "./artists-slider.css"
+
+export const ArtistsSlider: React.FunctionComponent<ArtistsSliderProps> = observer(({onSliderChange}) => {
+
+    const [recommendationLimit, setRecommendationLimit] = useState<number>(50);
```

Comment:
You can try using this value to control the initial slider value. Look up 'React controlled components'